toucan_connectors/google_big_query/google_big_query_connector.py

```diff
@@ -130,27 +192,30 @@ def _connect(credentials: Credentials) -> bigquery.Client:
         )
         return client
 
+    @staticmethod
+    def _clean_query(query: str) -> str:
+        # Ugly but the select query generated by frontend adds '"'
+        query = query.replace('"', '`')
+
+        # Since the interpolated variable is given the exact value and we
+        # don't want to change the old way clients were handling this in a
+        # query, we need to remove those surrounding single quotes for GBQ
+        # not considering it as a given string.
+        # \'@__QUERY_PARAM_0__\' => @__QUERY_PARAM_0__
+        return query.replace("\'@__", '@__').replace("__\'", '__')
```

Comment:
interesting 😄

Reply:
🙈 just a code moved from one place to another, let me update that a little bit

Reply:
[`1754923` (#1287)](https://github.com/ToucanToco/toucan-connectors/pull/1287/commits/17549233fad20b8d7233ad7aa571542ea9cf533d)